src/fidesops/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -227,6 +266,7 @@ def create_privacy_request(
 def get_request_status(
     *,
     db: Session = Depends(deps.get_db),
+    cache: FidesopsRedis = Depends(deps.get_cache),
```

Comment:
Is this cache being used implicitly somewhere in this function? At first glance it looks like it's not being used.

Reply:
good catch, it was being used in an early draft!